examples/analyze.py

```diff
@@ -100,26 +111,32 @@
             di.shape[0],
             dv))
 
-    # Basic gating/trimming
-    data = [FlowCal.gate.start_end(di, num_start=250, num_end=100)
-            for di in data]
-    data = [FlowCal.gate.high_low(di, sc_channels) for di in data]
-    # Gating of fluorescence channels
-    data = [FlowCal.gate.high_low(di, mef_channels) for di in data]
     # Transform to Relative Fluorescence Units (rfi), commonly known as
     # arbitrary units (a.u.)
-    data_transf = [FlowCal.transform.to_rfi(di, sc_channels)
+    data_transf = [FlowCal.transform.to_rfi(di, sc_channels + mef_channels)
                    for di in data]
 
-    # Transform to MEF
+    # Transform specified fluorescence channels to MEF
     print("\nTransforming to MEF...")
     data_transf = [to_mef(di, mef_channels) for di in data_transf]
 
+    # Gate
+
+    # Remove first and last events. Transients in fluidics can make the
+    # first few and last events slightly different from the rest.
+    data_gated = [FlowCal.gate.start_end(di, num_start=250, num_end=100)
+            for di in data_transf]
+
+    # Remove saturating events in forward/side scatter and fluorescence
+    # channels.
+    data_gated = [FlowCal.gate.high_low(di, sc_channels + mef_channels)
+            for di in data_gated]
```

Comment:
Examples will be changed, so this does not matter too much. I would suggest moving this discussion to #198.